src/ctree-app/ctree-landing.html

```diff
@@ -27,41 +27,83 @@
-<link rel="import" href="../../bower_components/app-layout/app-toolbar/app-toolbar.html">
-<link rel="import" href="../../bower_components/iron-list/iron-list.html">
-<link rel="import" href="../../bower_components/iron-location/iron-location.html">
-<link rel="import" href="../../bower_components/iron-location/iron-query-params.html">
-<link rel="import" href="../../bower_components/iron-meta/iron-meta.html">
-<link rel="import" href="../../bower_components/iron-scroll-target-behavior/iron-scroll-target-behavior.html">
-<link rel="import" href="../../bower_components/iron-selector/iron-selector.html">
-<link rel="import" href="../../bower_components/paper-button/paper-button.html">
-<link rel="import" href="../../bower_components/paper-fab/paper-fab.html">
-<link rel="import" href="../../bower_components/paper-icon-button/paper-icon-button.html">
-<link rel="import" href="../../bower_components/paper-input/paper-input.html">
-<link rel="import" href="../../bower_components/paper-spinner/paper-spinner.html">
-<link rel="import" href="../../bower_components/paper-tabs/paper-tabs.html">
-<link rel="import" href="../../bower_components/paper-tooltip/paper-tooltip.html">
+<link rel="import" href="../../bower_components/polymer/polymer-element.html" />
+<link
+  rel="import"
+  href="../../bower_components/app-layout/app-drawer/app-drawer.html"
+/>
+<link
+  rel="import"
+  href="../../bower_components/app-layout/app-scroll-effects/app-scroll-effects.html"
+/>
+<link
+  rel="import"
+  href="../../bower_components/app-layout/app-header/app-header.html"
+/>
+<link
+  rel="import"
+  href="../../bower_components/app-layout/app-header-layout/app-header-layout.html"
+/>
+<link
+  rel="import"
+  href="../../bower_components/app-layout/app-toolbar/app-toolbar.html"
+/>
+<link rel="import" href="../../bower_components/iron-list/iron-list.html" />
+<link
+  rel="import"
+  href="../../bower_components/iron-location/iron-location.html"
+/>
+<link
+  rel="import"
+  href="../../bower_components/iron-location/iron-query-params.html"
+/>
+<link rel="import" href="../../bower_components/iron-meta/iron-meta.html" />
+<link
+  rel="import"
+  href="../../bower_components/iron-scroll-target-behavior/iron-scroll-target-behavior.html"
+/>
+<link
+  rel="import"
+  href="../../bower_components/iron-selector/iron-selector.html"
+/>
+<link
+  rel="import"
+  href="../../bower_components/paper-button/paper-button.html"
+/>
+<link rel="import" href="../../bower_components/paper-fab/paper-fab.html" />
+<link
+  rel="import"
+  href="../../bower_components/paper-icon-button/paper-icon-button.html"
+/>
+<link rel="import" href="../../bower_components/paper-input/paper-input.html" />
+<link
+  rel="import"
+  href="../../bower_components/paper-spinner/paper-spinner.html"
+/>
+<link rel="import" href="../../bower_components/paper-tabs/paper-tabs.html" />
+<link
+  rel="import"
+  href="../../bower_components/paper-tooltip/paper-tooltip.html"
+/>
```

Comment:
This may have been an auto-reformat, but we want to keep them as the old style, as it's the standard convention for Polymer